modules/scanning-images-by-using-secured-clusters.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="scanning-images-by-using-secured-clusters_{context}"]
+= Scanning images by using secured clusters
+
+By configuring delegated image scanning, you can scan images by using the secured clusters instead of the Central services.
+
+A new delegated registry configuration specifies the registries from which you can delegate image scans. For images that Sensor observes, you can use the delegated registry configuration to delegate scans from no registries, all registries, or specific registries. 
+
+To enable delegation of scans by using the `roxctl` CLI, Jenkins plugin, or API, you must also specify a destination cluster and source registry.
+
+.Prerequisites
+
+* You have installed Scanner-slim in the secured cluster to scan images.
++
+[NOTE]
+====
+Enabling Scanner-slim is supported on {ocp} and Kubernetes secured clusters.
+====
+
+* You have set up image mirroring for registries.
++
+For more information, see "Enabling seamless image access and efficient use of multiple registry sources".
```

Comment:
Setting up image mirroring is not a pre-req for using delegated scanning, the two are not mutually exclusive. Recommend removing this line item from the pre-reqs.

Delegated scanning will scan images via the secured cluster. IF the user happens to have mirroring configured, delegated scanning will automatically attempt to scan images from the mirror references, however mirroring is NOT required to be setup to use delegated scanning, and in most user environments mirroring is NOT setup.